spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/TaskExecutionControllerTests.java

```diff
@@ -637,6 +637,13 @@ void sorting() throws Exception {
 				.andDo(print())
 				.andExpect(status().isOk());
 
+		mockMvc.perform(get("/tasks/executions").param("sort", "SCHEMA_TARGET").accept(MediaType.APPLICATION_JSON))
```

Comment:
This is good that we have added a controller test for this, but do we have anything that actually tests the query and verifies the sort works properly? That is the part that I am more concerned with.

Reply:
Added to `org.springframework.cloud.dataflow.server.repository.TaskExecutionExplorerTests`